XU-TechnicalPaper.md

```diff
@@ -25,163 +25,176 @@
+The idea of payment channels is around for a while, even Satoshi already explored [*high-frequency transactions*](https://en.bitcoin.it/wiki/Payment_channels). Then, with bidirectional channels, the inception of today's **Lightning Network** by Joseph Poon and Tadge Dryja, payment channels became more widely known. In short, payment channels enable instant, off-chain transfer of a digital token or coin, like bitcoin, directly between *two* parties. Payment channels are opened and closed via one on-chain transaction. A payment channel uses unbroadcast, but signed and fully valid on-chain transactions, which are sent back and forth between the two parties to do transactions. Since these unbroadcast transactions are merely data packets which are send via the internet layer and don't need to be validated on a blockchain, transfers are as fast as the internet connection between the two parties.
 
-Through their sophisticated setup, payment channels are trustless even though only the opening and closing transaction are included in the underlying blockchain. This is because intermediary transactions are signed, thus valid and can be broadcast at any time by either party. Payment channels also allow secure, trustless routing of funds through intermediaries using a technology called [Hashed Timelock Contracts (HTLCs)](https://en.bitcoin.it/wiki/Hashed_Timelock_Contracts), thus a direct channel between two parties is not required and a path through other XUDs can be used. This is described in detail in the [Lightning White Paper](https://lightning.network/lightning-network-paper.pdf). This is useful since even a single payment channel to a well-connected XUD could be enough to reach the rest of the network. Setting up payment channels can be compared to the act of transferring coins into a sidechain - coins have to be committed to a payment channel via a native on-chain transaction.
+Transactions are held on each side of the channel, in our case on the exchange’s XUD, representing the balance of each party. They use a multi-signature address as their input (the funding address) and point at two different addresses for their output, each controlled by one exchange. Setting up payment channels can be compared to transferring coins into a sidechain - coins have to be committed to a payment channel via a native on-chain transaction. This transaction 'locks' coins on the underlying chain and 'unlocks' them in the channel.
```

Comment:
It's technically not xud that's holding the unbroadcast transactions, so it might be worth rewording this.

Also I don't think that sidechains are well understood, so it's not a very helpful analogy, maybe it'd be best to just drop that sentence.

Reply:
Fixed, check if that's better.